File: benchmarks/suite5/prime.go
package main


func main() {
	const limit int64 = 100000
	var count int64
	for i := int64(2); i <= limit; i++ {
		prime := int64(1)
		for j := int64(2); j*j <= i; j++ {
			if i%j == 0 {
				prime = 0
				break
			}
		}
		if prime != 0 {
			count++
		}
	}
	suite5PrintI64(count)
}

File: benchmarks/suite5/prime.rs
mod bench_runtime;
use bench_runtime::suite5_print_i64;

fn main() {
    let limit: i64 = 100_000;
    let mut count: i64 = 0;
    for i in 2..=limit {
        let mut prime = 1i64;
        let mut j = 2i64;
        while j * j <= i {
            if i % j == 0 {
                prime = 0;
                break;
            }
            j += 1;
        }
        if prime != 0 {
            count += 1;
        }
    }
    suite5_print_i64(count);
}
